lib/ansible/modules/user.py

```diff
@@ -2730,6 +2730,27 @@ class BusyBox(User):
         - modify_user()
     """
 
+    # This can probably just move out to User, it might be useful in
+    # other strategies some day.
+    def _disable_password_in_shadow_or_fail(self, username):
+        try:
+            self.backup_shadow()
+            out_lines = []
+            with open(self.SHADOWFILE, 'r') as f:
+                for line in f.readlines():
+                    components = line.split(':')
+                    if components[0] == self.name:
+                        components[1] = '*'
+                        out_lines.append(':'.join(components))
+                    else:
+                        out_lines.append(line)
+            with open(self.SHADOWFILE, 'w') as f:
+                f.writelines(out_lines)
```

Comment:
this should use atomic_writes (but given how containers normally mount the file, it should note that for them users might need a toggle to allow  'unsafe=true'